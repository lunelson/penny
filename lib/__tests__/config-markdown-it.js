const mdiOptions = {};

const mdi = require('../config-markdown-it.js')(mdiOptions);
const mdi2 = require('../config-markdown-it.js')(mdiOptions);

test('mdi only configs once', () => {
  expect(mdi).toEqual(mdi2);
});

test('block rendering', () => {
  expect(mdi.render('hello __world__')).toMatchSnapshot();
});

test('inline rendering', () => {
  expect(mdi.renderInline('hello __world__')).toMatchSnapshot();
});
