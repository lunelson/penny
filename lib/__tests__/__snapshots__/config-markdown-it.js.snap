// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`block rendering 1`] = `
"<p>hello <strong>world</strong></p>
"
`;

exports[`inline rendering 1`] = `"hello <strong>world</strong>"`;
